app/lib/database/appGroup.ts

```diff
@@ -1,11 +1,5 @@
-import { NativeModules } from 'react-native';
-
-import { isIOS } from '../methods/helpers';
+import { NativeModules, Platform } from 'react-native';
 
 const { AppGroup } = NativeModules;
 
-const appGroup: { path: string } = {
-	path: isIOS ? AppGroup.path : ''
-};
-
-export default appGroup;
+export const appGroupPath: string = Platform.OS === 'ios' ? AppGroup.path : '';
```

Comment:
```
if (__DEV__ && isIOS) {
	console.log(`📂 appGroupPath`);
}
```
Pls, can you add this?
Because this `appGroup` returns the folder location of the app inside the emulator's folder and we can use any DB tools to access the SQLite database.